components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/BuildScanDataFields.java

```diff
@@ -77,6 +77,6 @@ private static String summaryTotal(BuildScanData data, String avoidanceOutcome,
     }
 
     private static String formatDuration(Duration duration) {
-        return duration == null ? NO_VALUE : String.valueOf(duration.toMillis());
+        return duration == null ? NO_VALUE : FormattingUtils.formatDuration(duration);
```

Comment:
This is needed until we can remove the `null` checks. Until then I opted for keeping it simple, though it is redundant.